VMAccess/HandlerManifest.json

```diff
@@ -1,13 +1,14 @@
 [{
```

Comment:
these formattings don't look like `jq` to me. I get `[` and `{` on separate lines?